### DISCUSSION TITLE
I want to restore my restricted account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33488] [Other Feature Feedback, Questions, & Ideas]
I want to restore my restricted account

**Comment by ghost** [comment]
Can i restore it?

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by ghost** [reply]
Approximately how long does it take to contact GitHub?

**Reply by airtower-luna** [reply]
I don't know. Filling the form should be pretty quick, but can't know how long it'll take before they get back to you.
